crates/bevy_pbr/src/light/point_light.rs

```diff
@@ -51,6 +51,6 @@ impl Default for PointLight {
 }
 
 impl PointLight {
-    pub const DEFAULT_SHADOW_DEPTH_BIAS: f32 = 0.02;
+    pub const DEFAULT_SHADOW_DEPTH_BIAS: f32 = 0.08;
```

Comment:
Does this produce any Peter-panning?

Reply:
Technically no, from what I've seen, but there is a little bit of a gap in `3d_scene`:

<img width="962" alt="Screenshot 2024-04-09 100610" src="https://github.com/bevyengine/bevy/assets/157897/832c4aff-6c61-487e-a48d-91d298f625b0">

Let me know what you'd like me to do: (1) adjust the default bias down and add the bias back in `3d_shapes`, or (2) leave as is.

Reply:
That looks good enough to me.